share/zm-testagent.lsb

```diff
@@ -20,6 +20,8 @@ PIDFILE=${ZM_BACKEND_PIDFILE:-/var/run/zonemaster/zm-testagent.pid}
 USER=${ZM_BACKEND_USER:-zonemaster}
 GROUP=${ZM_BACKEND_GROUP:-zonemaster}
 
+#export ZM_BACKEND_TESTAGENT_LOGLEVEL='warning'  # Set this variable to override the default log level
```

Comment:
`ZM_BACKEND_TESTAGENT_LOGLEVEL` is not used in `zonemaster_backend_testagent`. Maybe an omission.

Furthermore this might be in conflict with the option `--loglevel`.

Reply:
Good spotting! I made an update to make sure the log level value is propagated.